modules/storage/openstack.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+const (
+	// OpenStack is the global ServiceType that refers to all the components deployed
+	// by the meta-operator
+	OpenStack ServiceType = "Openstack"
```

Comment:
Yes, I've just imagined a world where `meta-operator` can propagate volumes to the services it orchestrates: this is supposed to be used at that level in case we don't want to specify each service:

```
Propagation:
- Glance
- Cinder
- Compute
```
vs: 

```
Propagation:
- OpenStack
```

but in the long term vision it can be different by `Everywhere` to cover multiple use cases.
If it helps, we can keep it simple and remove it for now, as long as the whole `openstack.go` that can be moved to the meta-operator and get versioned along with the defined API.

Reply:
+1 yes, not used atm, I'm fine removing it as `All` is basically the same thing atm.
When we have a clear idea of the orchestration logic, we can improve the propagation related filters accordingly.